bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingAddOp.java

```diff
@@ -188,10 +188,8 @@ void unsetSuccessAndSendWriteRequest(int bookieIndex) {
             return;
         }
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("Unsetting success for ledger: " + lh.ledgerId + " entry: " + entryId + " bookie index: "
+        LOG.info("Unsetting success for ledger: " + lh.ledgerId + " entry: " + entryId + " bookie index: "
```

Comment:
I don't think we should change this to `LOG.info`. It can be very annoying.

If you want this information, you can change your log4j properties to log debug statements for `PendingAddOp`.

Reply:
I thought this is helpful. If you don't agree I will revert it.